community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    alm-examples: |-
+      [{
+       "apiVersion": "hazelcast.com/v1alpha1",
+       "kind": "Hazelcast",
+       "metadata": {
+          "name": "hz"
+       },
+       "spec": {
+          "image": {
+            "tag": "3.11.2"
+          },
+          "hazelcast": {
+            "licenseKeySecretName": "hz-license-key"
```

Comment:
It's just a secret name (as described in the README). I think we should suggest to use secrets for the license key (instead of the plain text in the configuration file).

Reply:
ah ok. I did not realize that. I thought it is license key when I see `hz-license-key`. `hz-license-key-secret-name` might be better.

Reply:
Changed to `hz-license-key-secret`